json-simple/src/test/java/org/json/simple/Test.java

```diff
@@ -96,6 +96,7 @@ public void testDecode() throws Exception{
 			assertEquals(24, pe.getPosition());
 			System.out.println("Error at character position: " + pe.getPosition());
 			switch(pe.getErrorType()){
+			default:

```

Comment:
What's the functionality of the default case here?

Reply:
The default case functionality is to prevent unexpected bugs and handle all types of _pe_'s ErrorTypes (if we implement a new ErrorType it could be very useful). 
Unfortunately I forgot to add the `break` statement so I'm going to correct this.